elastic/kibana
# [Ingest Pipelines] Add ignore_missing_pipeline field to Pipeline processor

Closes https://github.com/elastic/kibana/issues/153208

## Summary

This PR adds the ignore_missing_pipeline field to the Pipeline processor form.

<img width="715" alt="Screenshot 2023-06-05 at 15 10 07" src="https://github.com/elastic/kibana/assets/59341489/77aedfa4-349b-4bde-ab62-4295cb08b653">


**How to test:**
1. Go to Stack Management -> Ingest Pipelines
3. Start creating a pipeline with one processor of type Pipeline - set the Pipeline name field to "missing-pipeline" and leave the `ignore_missing_pipeline` field to `false`.
4. After adding the processor, verify the request to Es by clicking the "Show request" button:
```
PUT _ingest/pipeline/<pipelineName>
{
  "processors": [
    {
      "pipeline": {
        "name": "missing-pipeline"
      }
    }
  ]
}
```
6. Then click on the "Add documents" button next to "Test Pipeline" and verify that the pipeline is always failing because there is no pipeline with the name "missing-pipeline".
7. Edit the processor and set the `ignore_missing_pipeline` field to `true`.
8. Verify the request:
```
PUT _ingest/pipeline/<pipelineName>
{
  "processors": [
    {
      "pipeline": {
        "name": "missing-pipeline",
        "ignore_missing_pipeline": true
      }
    }
  ]
}
```
10. Test the pipeline and verify that the pipeline doesn't fail because the `ignore_missing_pipeline` field is set to `true`.

### Checklist
